README_ES.md

```
@@ -12,82 +12,370 @@ Esta es una lista curada de **Free Experience Apps** — apps completamente grat
+  - [Saver Tuner](#saver-tuner)
+  - [Extinguish](#extinguish)
+  - [DarQ](#darq)
+  - [Oxygen Customizer](#oxygen-customizer)
+  - [EnforceDoze](#enforcedoze)
+  - [FlorisBoard](#florisboard)
+  - [MMRL](#mmrl)
+  - [Droid-ify](#droid-ify)
+- [Privacidad & Seguridad](#privacidad--seguridad)
+  - [Hypatia](#hypatia)
+  - [AdAway](#adaway)
+  - [XposedSmsCode](#xposedsmsCode)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid
Expected: #xposedsmscode; Actual: #xposedsmsCode

(MD051, link-fragments)